data/korath/korath ships.txt

```diff
@@ -8,11 +8,109 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+
+ship "Korath Dredger"
+	sprite "ship/dredger"
+	thumbnail "thumbnail/dredger"
+	attributes
+		category "Heavy Freighter"
+		"cost" 29370000
+		"shields" 39000
+		"hull" 16000
+		"required crew" 191
+		"bunks" 395
+		"mass" 1503
+		"drag" 18.3
+		"heat dissipation" .65
+		"fuel capacity" 900
+		"cargo space" 605
+		"outfit space" 827
+		"weapon capacity" 297
+		"engine capacity" 181
+		
+		weapon
+			"blast radius" 300
+			"shield damage" 4600
+			"hull damage" 2300
+			"hit force" 5900
+	outfits
+		"Korath Banisher" 2
+		"Korath Grab-Strike" 3
+		"Korath Warder" 2
+		
+		"Systems Core (Large)"
+		"Triple Plasma Core"
+		"Large Heat Shunt" 3
+		"Fuel Processor"
+		"Korath Repeater Rifle" 190
+		
+		"Steering (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Thruster (Asteroid Class)"
+		"Hyperdrive"
+		"Jump Drive"
+		
+	turret -37 -155 "Korath Banisher"
+	turret 0 -155 "Korath Warder"
+	turret 37 -155 "Korath Grab-Strike"
+	turret -96 0 "Korath Grab-Strike"
+	turret 96 0 "Korath Grab-Strike"
+	turret -80 130 "Korath Warder"
+	turret 80 130 "Korath Banisher"
+	engine -33 171 .4
+	engine -2 182 .6
+	engine 31 171 .4
+	fighter 51 59
+	fighter -51 59
+	fighter 0 -42
+	fighter 0 16
+	explode "tiny explosion" 120
+	explode "small explosion" 60
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "After the Korath were exiled, efficient resource collection became a priority, particularly through raiding. Their world-ships were too precious to risk on dangerous forays, and Raiders lacked the capacity. The Dredger was their first new ship: abundant cargo and passenger space, bays for four fighters, and seven turrets. With such a complement, these bulky ships are formidable supports for frontline Raiders."
```

Comment:
```suggestion
	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and Raiders lacked the cargo capacity. The Dredger was born as an answer to this problem, featuring an abundance of cargo and passenger space, with formidable defensive armament. With such a complement, these bulky ships can also offer potent support for frontline Raiders."
```

(and change the other description to match, of course :) )

Reply:
Looks good go me. Thanks!